scripts/post-build.js

```diff
@@ -0,0 +1,13 @@
+var fs = require('fs');
+
+configPath = `${__dirname.replace('scripts', 'config.ts')}`;
+
+fs.unlink(configPath, function (err) {
+    if (err) { throw err }
+    else {
+        fs.rename(`${configPath}_`, configPath, function (err) {
+            if (err) throw err
+            console.log('Original config brought back.')
+        });
+    }
+});
```

Comment:
I'll create a seperate issue to remove `var` from here and see what we can do with eslint to make this file not throw linting errors.

Reply:
created issue: https://github.com/hoprnet/hopr-admin/issues/208